core/graph/builders/run_graph_build.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Run the complete graph build with GPU optimization
+
+cd /home/todd/olympus/HADES-Lab/core/graph/builders
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)